dnngior/__init__.py

```diff
@@ -1,7 +1,7 @@
 __version__ = "0.0.3"
 
 import os
-os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' 
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '5'
```

Comment:
Is tensorflow now a strong dependency ? if it is optional, then i think this may be better to be under an if..else statement because I guess it would lead to an error.